.vscode/settings.json

```diff
@@ -22,7 +22,11 @@
   "[markdown]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.formatOnSave": true,
-    "editor.quickSuggestions": true,
+    "editor.quickSuggestions": {
+      "comments": "on",
+      "strings": "on",
+      "other": "on"
+    },
```

Comment:
what does this change?

Reply:
It must have been added automatically by VSCode.